test/lib/testMayaUsdExportPreviewShader.py

```diff
@@ -0,0 +1,63 @@
+#!/usr/bin/env python
+
+#
+# Copyright 2020 Apple
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+import os
+import unittest
+import tempfile
+from maya import cmds
+from pxr import Usd
+
+import logging
+logging.basicConfig()
+logger = logging.getLogger()
+logger.setLevel(logging.DEBUG)
+
+def apply_preview_surface(node, shader=None):
+    """Apply a USD Preview Surface shader to the given node"""
+    shader = shader or cmds.shadingNode('usdPreviewSurface', asShader=True)
+    cmds.select(node)
+    cmds.hyperShade(assign=shader)
+    return shader
+
+class MayaUsdExportPreviewSurface(unittest.TestCase):
+
+    @classmethod
+    def setUpClass(cls):
+        cmds.loadPlugin('mayaUsdPlugin')
+        cls.temp_dir = tempfile.mkdtemp()
```

Comment:
You might want to consider moving this test to test/lib/usd/translators 
In that folder there are utilities that will help setup a full test environments in the build area where the output files will be preserved, allowing easy investigation in case of problems. Look for `inputPath = fixturesUtils.setUpClass(__file__)` in files like https://github.com/Autodesk/maya-usd/blob/dev/test/lib/usd/translators/testUsdExportRfMShaders.py

Reply:
@JGamache-autodesk Thanks! I'll switch over our tests to live there. Quick question regarding naming...
I've been calling our tests `testMayaUsdxxx` but the ones in the translator section are `testUsdxxx` or `textUsdMayaxxx`. Which naming should I be going with?

Reply:
We haven't really set any naming convention for tests other than it should start with `test`. For translators, please follow the naming convention you see in` https://github.com/Autodesk/maya-usd/blob/dev/test/lib/usd/translators/`, i.e. `testUsdxxx`.

Reply:
Okay moved this over.